eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -876,7 +876,7 @@ stages:
             crossRootFs: '/crossrootfs/x64'
             targetOS: linux-musl
             targetArchitecture: x64
-            extraProperties: /p:TargetRID=linux-musl-x64
+            targetRid: ${{ variables.alpinelinuxX64Rid }}
```

Comment:
All these `alpinelinux` RID references need to be updated to `linuxMusl`.

Reply:
Thanks for the check, updated the code.